File: Piko/Sources/SFBreakpoint.m
//
//  SFBreakpoint.m
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

#import "SFBreakpoint.h"

@implementation SFBreakpoint

+ (instancetype)breakpointWithTime:(double)time value:(double)value
{
    return [[SFBreakpoint alloc] initWithTime:time value:value];
}

- (instancetype)initWithTime:(double)time value:(double)value
{
    self = [super init];
    if (self != nil) {
        _time = time;
        _value = value;
    }
    return self;
}

@end

File: Piko/Sources/SFBreakpoint.swift
//
//  SFBreakpoint.swift
//
//  Copyright (c) 2014 Satoshi Muraki. All rights reserved.
//

import Foundation

@objc class SFBreakpoint: NSObject {

    let time: Double
    let value: Double

    init(time: Double, value: Double) {
        self.time = time
        self.value = value
        super.init()
    }
}
